### PR TITLE
Classification search fix [AtlasProxy]

### PR DESCRIPTION
**Pull request by mgorsk1**
### Summary of Changes

Following latest changes in search & atlas, this PR makes classification search fully functional.

### Tests

_What tests did you add or modify and why? If no tests were added or modified, explain why. Remove this line_

### Documentation

_What documentation did you add or modify and why? Add any relevant links then remove this line_

### CheckList
Make sure you have checked **all** steps below to ensure a timely review.
- [x] PR title addresses the issue accurately and concisely. Example: "Updates the version of Flask to v1.0.2"
    - In case you are adding a dependency, check if the license complies with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
- [x] PR includes a summary of changes. 
- [x] PR adds unit tests, updates existing unit tests, __OR__ documents why no test additions or modifications are needed.
- [x] In case of new functionality, my PR adds documentation that describes how to use it.
    - All the public functions and the classes in the PR contain docstrings that explain what it does
- [x] PR passes `make test`

### CODE REVIEW

**Comment by mgorsk1** [comment]
cc @verdan

File: search_service/proxy/atlas.py [hunk 1]
```diff
@@ -256,18 +255,13 @@ def fetch_table_search_results_with_filter(self, *,
...
-
         # Currently Atlas doesn't allow mixing search by filters and classifications
         if filters:
             query_params = self._prepare_basic_search_query(self.page_size, page_index,
                                                             filters=filters)
-        elif classifications:
+        elif tags_filter_value:
             query_params = self._prepare_basic_search_query(self.page_size, page_index,
-                                                            classifications=classifications)
+                                                            classification=tags_filter_value[0])
```
Comment (verdan) [inline search_service/proxy/atlas.py:264]: @mgorsk1  so `tags_filter_value` is a list coming from frontend service ? Can we filter based on multiple tags at the same time ?

Comment (mgorsk1) [reply search_service/proxy/atlas.py:264]: that's what I was hoping for but unfortunately no - the only way would be to make several queries and join results, but that would mess up interpretation of popularityScore

Comment (dwarszawski) [reply search_service/proxy/atlas.py:264]: looks good to me.

**Pull request merged**